aap-common/proc-aap-pull-command-container-image.adoc

```diff
@@ -4,12 +4,22 @@
 
 Pull the Docker image for the Ansible on Clouds operational container with the same tag version as your deployment.
 
-For example, if your foundation deployment version is 2.2.20230215, you must pull the operational image with tag 2.2.20230221-00.
+[NOTE]
+====
+Before Pulling the docker image, make sure you are logged in to redhat registry using docker. Use the following command to login to registry.redhat.com.
```

Comment:
For "logged in to redhat registry ", should "redhat" be "Red Hat" in these cases?